### PR TITLE
WMI::IsBitlockedDrive: check ConversionStatus rather than ProtectionStatus

### PR DESCRIPTION
**Pull request by ramcq**
A drive can be encrypted (partially or fully) without the key being sealed by the TPM which makes it protected. Conversion status tells us whether the bytes on disk are in clear or not.

https://phabricator.endlessm.com/T21417

### CODE REVIEW

**Comment by ramcq** [comment]
Untested bilge. Let's see what Beatriz finds.

File: src/endless/WMI.cpp [hunk 1]
```diff
@@ -102,7 +102,7 @@ BOOL WMI::IsBitlockedDrive(const CString & drive)
     const CString objectPath(L"ROOT\\CIMV2\\Security\\MicrosoftVolumeEncryption");
     CString bitlockerQuery;
 
-    bitlockerQuery.Format(L"Select * from Win32_EncryptableVolume where ProtectionStatus != 0 and DriveLetter = '%ls'", drive);
+    bitlockerQuery.Format(L"Select * from Win32_EncryptableVolume where ConversionStatus != 0 and DriveLetter = '%ls'", drive);
```
Comment (wjt) [inline src/endless/WMI.cpp]: Well spotted. Can we add a link to https://msdn.microsoft.com/en-us/library/windows/desktop/aa376483(v=vs.85).aspx in this function? And yes, let's wait and see what Beatriz comes back with.

**Comment by wjt** [comment]
@ramcq how's that?

**Comment by ramcq** [comment]
Looks legit.

**Pull request merged**